web/frontend/src/components/camera/camera.svelte

```diff
@@ -14,29 +15,26 @@ const { robotClient, streamManager } = useRobotClient();
 
 let imgEl: HTMLImageElement;
 
-let cameraFrameIntervalId = -1;
-
 const cameraManager = $streamManager.setCameraManager(cameraName);
 
-const clearFrameInterval = () => {
-  window.clearInterval(cameraFrameIntervalId);
-};
+let clearFrameInterval = noop;
```

Comment:
`noop` isn't defined anywhere within this script, this'll throw an error.

Typing it like this:
```suggestion
let clearFrameInterval: (() => Promise<void>) | undefined
```
And modifying all the calls with an optional chaining operator should be sufficient:
```
clearFrameInterval?.()
```

Reply:
I advised Abe import `noop` from `lodash`. This works too.

Reply:
That would require fewer changes, @abe-winter in that case this file is just missing a:
```ts
import { noop } from 'lodash-es';
```

Reply:
oops -- I added vanilla lodash dep instead of using lodash-es. I'll change over to `-es`

Reply:
ahh my bad, PEBCAK